mobilecoind/clients/python/templates/block.html

```diff
@@ -0,0 +1,69 @@
+<html>
+    <head>
+        <title>MobileCoin Blockchain Explorer</title>
+        <style>
+            td {text-align: left; padding-right: 20px;}
+            body {font-family: 'Courier New', Courier, monospace; background-color: black; color: #00FF00;}
+        </style>
+    </head>
+    <body>
+        <pre>
+********************************************************************************
+                       MobileCoin Blockchain Explorer
+                                 Block {{ block_num }}
+********************************************************************************
+Size (txos)  {{ txo_count }}
+Size (kB)    {{ size_of_block }}
+Hash         {{ block_hash.hex() }}
+Key Images   {{ key_image_count }}
+
+
+********************************************************************************
+Key Images
+********************************************************************************
+            <table>
+                <thead>
+                    <tr>
+                        <td>Index</td><td>Key Image (base64 bytes)</td>
+                    </tr>
+                    <tr>
+                      <td>*****</td><td>************************</td>
+                    </tr>
+                </thead>
+                <tbody>
+                    {%- for key_image in key_images %}
```

Comment:
`key_images` should just be a list, without being wrapped in `enumerate()`. Jinja2 templates support `loop.index` inside the loop to get the index - see https://jinja.palletsprojects.com/en/2.11.x/templates/#for